README.md

```diff
@@ -1 +1,195 @@
+>
+
+- **Adding food recipes**
+- **Sing up**
+- **Sing out**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to :
+
+- To have Ruby installed
+- To have Rails 7 installed also
+- Dependencies for basic gems that project uses
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/husush23/Recipe_app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+- For eslint run the following:
+
+```sh
+  rubocop
+```
+
+- For Stylelint(CSS) run the following:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+- For testig run the following:
+
+```sh
+  bundle exec rspec
+```
+
+### Deployment
+
+You can deploy this project using:
+
+- Heroku etc.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author 1**
+
+👤 **Hussein Hashi**
+
+- GitHub: [Husush23](https://github.com/husush23)
+- Twitter: [Hussein](https://twitter.com/HusseinKadare2)
+- LinkedIn: [Hussein](https://www.linkedin.com/in/hussein-kadare-8aa1361a8/)
+
+👤 **Author 2**
+
+👤 **Saloman**
+
+- GitHub: [Tshobohwa](https://github.com/Tshobohwa)
+- LinkedIn: [Tshobohwa](https://www.linkedin.com/in/tshobohwasalomon/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Single page app**
+- **Complete webApp**
+- **Smooth functionality(design)**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/husush23/Recipe_app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you encounter any issues or have questions about the project, feel free to reach out for support.Checkout teh socail links included in the authors, contact any of one or simply contribute this repo by raising a pull request on and always try to check the issues page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my heartfelt appreciation to Microverse for helping doing such projects, it's building our mindS and I am thankfully appreciating their efforts for heliping doing these tasks.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix the MIT license link as it is still broken.
![Screen Shot 2023-09-15 at 1 58 59 PM](https://github.com/husush23/Recipe_app/assets/62849872/bf004573-f036-4bc9-8829-547dc6933a19)